dashboard/test/ui/features/hiddenScripts_eyes.feature

```diff
@@ -0,0 +1,23 @@
+@eyes
+@dashboard_db_access
+@no_circle
```

Comment:
no_circle because in circle we dont seed any courses. I'm going to explore fixing that in another branch, but dont want to let that block getting this in.